elevate-cpanel

```diff
@@ -232,43 +232,134 @@ use constant GRUB_EDITENV      => '/usr/bin/grub2-editenv';
 use constant GRUB_ENV_FILE     => '/boot/grub2/grubenv';
 
 use constant DISABLE_MYSQL_YUM_REPOS => qw{
-  Mysql57.repo
-  Mysql80.repo
-
-  MariaDB102.repo
-  MariaDB103.repo
-  MariaDB105.repo
-  MariaDB106.repo
-
-  mysql-community.repo
-
+  mysql-connectors-community
+  mysql-tools-community
+  mysql55-community
+  mysql56-community
+  mysql57-community
+  mysql80-community
+  mysql-tools-preview
+  mysql-cluster-7.5-community
+  mysql-cluster-7.6-community
+  mysql-connectors-community-source
+  mysql-tools-community-source
+  mysql55-community-source
+  mysql56-community-source
+  mysql57-community-source
+  mysql80-community-source
+  mysql-tools-preview-source
+  mysql-cluster-7.5-community-source
+  mysql-cluster-7.6-community-source
+  mysql57-community-source
+  mysql80-community-source
+  mysql-connectors-community-source
+  mysql-tools-community-source
+  mysql-tools-preview-source
+  mysql-cluster-7.5-community-source
+  mysql-cluster-7.6-community-source
+  mysql-cluster-8.0-community-source
+  mysql57-community
+  mysql80-community
+  mysql-connectors-community
+  mysql-tools-community
+  mysql-tools-preview
+  mysql-cluster-7.5-community
+  mysql-cluster-7.6-community
+  mysql-cluster-8.0-community
+  mysql80-community-debuginfo
+  mysql-connectors-community-debuginfo
+  mysql-tools-community-debuginfo
+  mysql-cluster-8.0-community-debuginfo
+  MariaDB102
+  MariaDB103
+  MariaDB105
+  MariaDB106
 };
 
 use constant VETTED_YUM_REPO => qw{
-  CentOS-Base.repo
-  CentOS-CR.repo
-  CentOS-Debuginfo.repo
-  CentOS-fasttrack.repo
-  CentOS-Media.repo
-  CentOS-Sources.repo
-  CentOS-Vault.repo
-  CentOS-x86_64-kernel.repo
-  cPAddons.repo
-  cpanel-plugins.repo
-  cp-dev-tools.repo
-  EA4.repo
-  ELevate.repo
-  elasticsearch.repo
-  epel-testing.repo
-  epel.repo
-  imunify-rollout.repo
-  imunify360.repo
-  influxdb.repo
-  kernelcare.repo
-  wp-toolkit-cpanel.repo
-  imunify360-ea-php-hardened.repo
-  droplet-agent.repo
+  base
+  updates
+  extras
+  centosplus
+  cr
+  base-debuginfo
+  c7-media
+  base-source
+  updates-source
+  extras-source
+  centosplus-source
+  C7.0.1406-base
```

Comment:
Is this for cloudlinux? we don't support that yet.

Reply:
That was just names of stuff I pulled from the centos-vault repo. No idea why those all looked so odd.